Add Feature Highlights section to Open page

alex-ketch opened the pull request:
Key highlights are:
- Replaced the top level `Stencila /Open` headline with a "sales-pitch tagline". Felt it was redundant since we have the name in the hero nav. I tried to keep the subheading close to the intent of the original, without being repetitive of the headline. Right now it's a bit wordy, so need to bring out an axe to lop off some of it.
- Changed the tooltip `?` to be outlines instead of filled in, to better bring focus to the primary `Open` document button
- Added the `How can Stencila Open  help?` section

![Stencila-Open-draft](https://user-images.githubusercontent.com/1646307/65549238-20c56900-deeb-11e9-9892-2fdb06e4fec1.png)

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -1,12 +1,22 @@
-                <h1 class="title has-text-centered">Stencila /open</h1>
-                <h2 class="subtitle is-5 has-text-centered">Generate reproducible publication previews, and collaborate with colleagues using other formats.</h2>
+                <h1 class="title is-size-4-mobile has-text-centered">
+                  Reproducible research collaboration &amp; publication, minus&nbsp;all&nbsp;the&nbsp;headache.
```
"Reproducible research collaboration & publication, minus the headaches" (remove "all the").

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -1,12 +1,22 @@
+                <h2 class="subtitle is-5 has-text-centered">Generate previews, collaborate across file formats, and
+                  concentrate on getting your work done.
+                  <br />
+                  Need to share your Jupyter notebook with your supervisor as a Word document for&nbsp;sign&#8209;off?&nbsp;Easy&nbsp;peasy.
```
To make this shorter I think that "Generate previews, collaborate across file formats, and concentrate on getting your work done." could be removed. But not convinced it's necessary.

I think that using animated typing could be a way to keep the text short while showing the flexibility of the tool. Maybe a loop deleting and typing `["Jupyter Notebook on Github", "R Markdown document", "Markdown document on HackMD"]` etc or  `["Jupyter Notebook", "R Markdown document"]`.

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
+              </h3>
+
+              <p class="is-size-5">
+              Go back-and-forth across file formats as needed so you can efficiently collaborate
```
"Go" -> "Convert" ?

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
+          <div class="columns is-vcentered is-centered has-background-light">
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Preserve data across formats
```
"data" -> "your code"

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
+              </h3>
+
+              <p class="is-size-5">
+                By embedding the source-code inside generated figures, we are able to maintain reproducibility
```
"By embedding your source code inside generated figures and other outputs"

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
+              </p>
+
+              <p class="is-size-5">
+                The themes, like all our work, are open-source and can be changed based on your needs. 
```
Add "Create your own themes".

alex-ketch replied:
I changed the wording here, mind taking another look please @nokome?

beneboy commented on director/stencila_open/templates/open/main.html:
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
+
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Beautifully styled papers to match&nbsp;your&nbsp;needs
```
Personally I cringe every time a company describes their products as "beautiful". I find it extremely overused these days.

nokome replied:
I don't feel quite as strongly as Ben but do also hesitate a little with that wording. Plus, I don't really think the generated HTML  documents are always beautiful - it depends a lot on the content.

alex-ketch replied:
Mind suggesting an alternative please @beneboy?

beneboy replied:
Don't feel the need to change it just on my account, just my opinion. My first thought was "elegantly styled" but that's in the same vein, like come on… beautiful?, elegant? it's a freakin HTML page!

What about just "Papers style to match your needs"?

alex-ketch replied:
Thanks for the input everyone, helps guide the changes.
I changed it to `Multiple themes to style your manuscript` to keep it objective and highlight the fact that users have options.

nokome commented:
I feel that the examples are now too far down the page. Some options could be:

- make it a section below "Seamless file conversion".
- make the examples a horizontally scrollable carousel below the form (my preferred option; https://creativebulma.net/product/carousel/demo may be of use)

nokome commented on director/stencila_open/templates/open/main.html:
```diff
@@ -221,19 +219,18 @@ <h3 class="title is-size-4-mobile">
               <p class="is-size-5">
-              Go back-and-forth across file formats as needed so you can efficiently collaborate
-              with every stakeholder.
+                Conver back-and-forth across file formats as needed so you can efficiently collaborate with every
```
missing t

alex-ketch commented:
I looked into and [fixed](https://github.com/stencila/hub/pull/252/commits/b2221c2edbcc16bcb4a261aa22a642934c24b6dd) the issue we observed during the call yesterday, with custom components not being properly styled when changing the themes.
Was due to a hardcoded Thema version which didn't include the CSS variables. I've changed to be range `<=1` now.

---

> At a minimum I think the "Upload File" button should be solid (like the "Open" button). But I also think we could go a little further to put the upload functionality on more of an equal footing with opening a URL (currently if feels very secondary).

I also think having it all in one line would be the ideal solution, but should be done in another PR. 
Changing the alignment requires moving around some related pieces (tooltips, errors), and potentially changing the form setup, hence why I'd want to be more cautious with the changes.

For now I've made it solid to match the `Open` button. Please feel free to revert @jwijay.

> I think that we can iterate on the rPNG schematic some more to better capture the idea of embedding code in the data. But for now I think we should at least make the sample code more realistic e.g. plot(data) or plot(height~width, data)

I've changed the graphic based on your `plot(height~width, data)` example.

---

There are a few other suggestions made for this PR, but I'd like to explore them as separate PRs if no one objects.

1.
 > I feel that the examples are now too far down the page. Some options could be:
  > - make it a section below "Seamless file conversion".
  > - make the examples a horizontally scrollable carousel below the form (my preferred option; https://creativebulma.net/product/carousel/demo may be of use)

Agreed, but I think @beneboy is making changed to the examples section, so would defer to his PR for placement changes.

2. 
> To make this shorter I think that "Generate previews, collaborate across file formats, and concentrate on getting your work done." could be removed. But not convinced it's necessary.
>
>I think that using animated typing could be a way to keep the text short while showing the flexibility of the tool. Maybe a loop deleting and typing ["Jupyter Notebook on Github", "R Markdown document", "Markdown document on HackMD"] etc or ["Jupyter Notebook", "R Markdown document"].
